pom.xml

```diff
@@ -458,6 +458,7 @@
         <version.org.jboss.narayana>5.12.5.Final</version.org.jboss.narayana>
         <version.org.jboss.openjdk-orb>8.1.9.Final</version.org.jboss.openjdk-orb>
         <version.org.jboss.resteasy>4.7.4.Final</version.org.jboss.resteasy>
+        <version.org.jboss.resteasy.tracing.api>2.0.0.Alpha1</version.org.jboss.resteasy.tracing.api>
```

Comment:
We can't use this version here as it targets Jakarta REST 3.1. This version needs to be 1.0.0.Final here, but 2.0.0.Alpha1 in the `ee-9/pom.xml`. Doing it this way will allow us to develop the feature, while not having to worry about the Jakarta REST version being used. I apologized I didn't think of this sooner.

Reply:
@jamezp okay! I'll revert it back to `1.0.0.Final`.

Reply:
added `2.0.0.Alpha1` in `ee-9`: https://github.com/wildfly/wildfly/pull/15198/commits/1dca57ac1351df26bba661ffa48dd768b765ed4d

Reply:
After built dist locally it can verify that the `ee-9` dist and comment dist has their own `tracing-api` versions:

<img width="1920" alt="image" src="https://user-images.githubusercontent.com/201907/173629619-b755853d-6e49-4f13-a3f9-b2dbd45f1b8c.png">

Reply:
Do we need that version? IIRC there are no Jakarta EE dependencies on the module.

Reply:
I think we can use it in the main POM. Since it has no Jakarta EE 9+ dependencies, there is no need to put it in the `ee-9` dependencies. It could just go straight into the main dependencies where we currently have 1.0.0.Final defined.

Reply:
@jamezp Okay!